packages/api/endpoints/bulk-delete.js

```diff
@@ -17,25 +19,18 @@ async function startBulkDeleteAsyncOperation(req, res) {
     tableName: process.env.AsyncOperationsTable
   });
 
-  let asyncOperation;
-  try {
-    asyncOperation = await asyncOperationModel.start({
-      asyncOperationTaskDefinition: process.env.AsyncOperationTaskDefinition,
-      cluster: process.env.EcsCluster,
-      lambdaName: process.env.BulkDeleteLambda,
-      description: 'Bulk Delete',
-      operationType: 'Bulk Delete',
-      payload: { granuleIds: req.body.granuleIds }
-    });
-  } catch (err) {
-    if (err.name !== 'EcsStartTaskError') throw err;
-
-    return res.boom.serverUnavailable(`Failed to run ECS task: ${err.message}`);
-  }
+  const asyncOperation = await asyncOperationModel.start({
```

Comment:
There is no need to catch these errors anymore because if an error is thrown here then the `asyncOperationEndpointErrorHandler` middleware is automatically triggered and will return the appropriate response